js/react_native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java

```diff
@@ -104,38 +132,31 @@ public void run(String key, ReadableMap input, ReadableArray output, ReadableMap
    */
   public WritableMap loadModel(String uri, ReadableMap options) throws Exception {
     InputStream modelStream = reactContext.getApplicationContext().getContentResolver().openInputStream(Uri.parse(uri));
-    WritableMap resultMap = loadModel(uri, modelStream, options);
+
+    byte[] modelData = null;
+    Reader reader = new BufferedReader(new InputStreamReader(modelStream));
+    modelData = new byte[modelStream.available()];
+    modelStream.read(modelData);
     modelStream.close();
+    WritableMap resultMap = loadModel(modelData, options);
     return resultMap;
   }
 
   /**
-   * Load a model from raw resource directory.
+   * Load a model from buffer.
    *
-   * @param uri uri parameter from react native loadModel() or dummy string for unit testing purpose
-   * @param modelStream a input stream to read a model
+   * @param modelData the model data buffer
    * @param options onnxruntime session options
    * @return model loading information, such as key, input names, and output names
    */
-  public WritableMap loadModel(String uri, InputStream modelStream, ReadableMap options) throws Exception {
-    OrtSession ortSession = null;
-
-    if (!sessionMap.containsKey(uri)) {
```

Comment:
What's the reason for removing the check for re-loading the same model? 

Is a documentation change required for this as an app that was sloppy about how many times the model was loaded would now get duplicate inference sessions created. Not sure how likely that is in the real world though.

Reply:
As long as we support loading model from buffer, it is no longer possible to use the model's filepath as key. so I have to remove this check.

There is no documentation for this, and the duplication is never guaranteed - we don't do this for any language bindings for ORT. If users create inference session twice on the same path, we create 2 instances, and this is the expectation.